app/src/main/java/org/kiwix/kiwixmobile/KiwixMobileActivity.java

```diff
@@ -886,6 +896,52 @@ private void goToBookmarks() {
     startActivityForResult(intentBookmarks, BOOKMARK_CHOSEN_REQUEST);
   }
 
+  private void gotoTextChangeDialog(){
+    Dialog yourDialog = new Dialog(this);
+    LayoutInflater inflater = (LayoutInflater)this.getSystemService(LAYOUT_INFLATER_SERVICE);
+    View layout = inflater.inflate(R.layout.dialogbox_seekbar, (ViewGroup)findViewById(R.id.dialog));
+    yourDialog.setContentView(layout);
+    yourDialog.show();
+    Window window = yourDialog.getWindow();
+    window.setLayout(900, 300);
+    SeekBar simpleSeekBar = (SeekBar)layout.findViewById(R.id.your_dialog_seekbar);
+    simpleSeekBar.setMax(3);
+    simpleSeekBar.setProgress(2);
+
+    simpleSeekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
+      @Override
+      public void onProgressChanged(SeekBar seekBar, int i, boolean b) {
+        WebSettings webSettings = getCurrentWebView().getSettings();
+        switch (i){
+          case 0:
+            webSettings.setTextSize(WebSettings.TextSize.SMALLEST);
+            break;
+          case 1:
+            webSettings.setTextSize(WebSettings.TextSize.SMALLER);
+            break;
+          case 2:
+            webSettings.setTextSize(WebSettings.TextSize.NORMAL);
+            break;
+          case 3:
+            webSettings.setTextSize(WebSettings.TextSize.LARGER);
+            break;
+        }
+
+      }
+
+      @Override
+      public void onStartTrackingTouch(SeekBar seekBar) {
+
```

Comment:
Please verify that keeping these methods empty won't be a problem. Even if doesn't, please provide a log in both the methods, i.e., `onStartTrackingTouch` and `onStopTrackingTouch` for future reference.

Reply:
@brijeshshah13  So you want a Log.d kind of a message or just a comment?

Reply:
@harshvardhan-takawale Log.d kindof message. Please refer some other files to see how we do logging.